nuxeo-features/nuxeo-platform-oauth/src/main/java/org/nuxeo/ecm/platform/oauth2/providers/NuxeoOAuth2ServiceProvider.java

```diff
@@ -65,10 +65,14 @@
 
     protected Long id;
 
+    private String description;
```

Comment:
Never use `private`. I know it's the style of the other fields in this class (which you don't need to clean up right now), but for new fields just use `protected`.